samcli/settings/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 import os
 
-if "__SAM_CLI_TELEMETRY_ENDPOINT_URL" not in os.environ:
-    telemetry_endpoint_url = "https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics"
-else:
-    telemetry_endpoint_url = os.getenv("__SAM_CLI_TELEMETRY_ENDPOINT_URL")
+
+telemetry_endpoint_url = os.getenv(
+    "__SAM_CLI_TELEMETRY_ENDPOINT_URL", "https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics"
+)
```

Comment:
this change was addressing inconsistent types:

- above: `str`
- below: `Optional[str]`